.github/workflows/builds.yml

```diff
@@ -39,6 +36,13 @@ jobs:
         cd shacl
         python generate_shacl.py
         cd ..
+    - name: upload ttls
+      if: ${{ matrix.python-version }} == "3.7.9"
+      uses: actions/upload-artifact@v3
+      with:
+        name: ontology
```

Comment:
Can we change the name of the artifact to "Brick-files" or something more descriptive and Brick-specific than "ontology"? "Brick-ontology" would be fine as well